apis/run/v1alpha2/groupversion_info.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Q: Did TKGS already release a version supporting v1alpha2? I am asking to know if the API types are finalized or if there is any scope for change.